utils/mil_tools/mil_ros_tools/image_helpers.py

```diff
@@ -121,3 +122,103 @@ def convert(self, data):
+        self.camera_info_right = None
+        self.image_left = None
+        self.image_time_left = None
+        self.image_right = None
+        self.image_time_right = None
+
+        # Subscribe to image and camera info topics for both cameras
+        root_topic_left, image_subtopic_left = path.split(left_image_topic)
+        self._info_sub_left = rospy.Subscriber(
+            root_topic_left + '/camera_info', CameraInfo,
+            lambda info: setattr(self, 'camera_info_left', info), queue_size=queue_size)
+        image_sub_left = message_filters.Subscriber(left_image_topic, Image)
+        root_topic_right, image_subtopic_right = path.split(right_image_topic)
+        self._info_sub_right = rospy.Subscriber(
+            root_topic_right + '/camera_info', CameraInfo,
+            lambda info: setattr(self, 'camera_info_right', info), queue_size=queue_size)
+        image_sub_right = message_filters.Subscriber(right_image_topic, Image)
+
+        # Use message_filters library to set up synchronized subscriber to both image topics
+        if slop is None:
+            self._image_sub = message_filters.TimeSynchronizer([image_sub_left, image_sub_right], queue_size)
+        else:
+            self._image_sub = message_filters.ApproximateTimeSynchronizer([image_sub_left, image_sub_right],
+                                                                          slop, queue_size)
+        self._image_sub.registerCallback(self._image_callback)
+
+    def wait_for_camera_info(self, timeout=10, unregister=True):
+        '''
+        Blocks until camera info has been received.
+
+        @param timeout Time in seconds to wait before throwing exception if camera info is not received
+        @param unregister Boolean, if True will unsubscribe to camera info after receiving initial info message,
+               so self.camera_info_left and self.camera_info_right will not be updated
+        @return Tuple(camera_info_left, camera_info_right) camera info for each camera if received before timeout
+        @throws Exception if camera info for both cameras is not received within timeout
+        '''
+        timeout = rospy.Time.now() + rospy.Duration(timeout)
+        while (rospy.Time.now() < timeout) and (not rospy.is_shutdown()):
+            if self.camera_info_left is not None and self.camera_info_right is not None:
+                if unregister:
+                    self._info_sub_left.unregister()
+                    self._info_sub_right.unregister()
+                return self.camera_info_left, self.camera_info_right
+            rospy.sleep(0.05)
+        if self.camera_info_left is not None and self.camera_info_right is not None:
+            if unregister:
+                self._info_sub_left.unregister()
+                self._info_sub_right.unregister()
+            return self.camera_info_left, self.camera_info_right
+        raise Exception("Camera info not found.")
+
+    def _image_callback(self, left, right):
```

Comment:
left_img and right_img would be more immediately parseable. Else, I'd add it to the docstring.